client/components/SurrenderForm.js

```diff
@@ -0,0 +1,188 @@
+import React, { useState } from "react"
+import ErrorList from "./ErrorsList"
+
+const SurrenderForm = (props) => {
+  const [message, setMessage] = useState("")
+  const [errors, setErrors] = useState({})
+  const [surrender, setSurrender] = useState({
+    personName: "",
+    phoneNumber: "",
+    email: "",
+    petName: "",
+    age: "",
+    petTypeId: "",
+    imgUrl: "",
+    vaccinationStatus: "",
+    adoptionStory: "",
+  })
+
+  const handleInputChange = (event) => {
+
+    setSurrender({
+      ...surrender,
+      [event.currentTarget.id]: event.currentTarget.value,
+    })
+  }
+
+  const submitHandler = (event) => {
+    event.preventDefault()
+    if (isValid()) {
+      addPetForSurrender()
+    } else {
+      console.log("form Not submitted!")

```

Comment:
Is this the proper error message? the form was submitted, it's that the input wasn't valid, right?

Reply:
Great catch!  We updated to "Inputs not valid!"